docs/maui/converters/is-in-range-converter.md

```diff
@@ -0,0 +1,120 @@
+---
+title: IsInRangeConverter - .NET MAUI Community Toolkit
+author: GeorgeLeithead
+description: "The IsInRangeConverter is a one-way converter that takes an incoming value implementing IComparable, and a minimum and maximum value, and returns the result of the value being between the minimum and maximum values."
+ms.date: 11/26/2022
+---
+
+# IsInRangeConverter
+
+The `IsInRangeConverter` is a one-way converter that takes an incoming value implementing `IComparable`, and returns the result of the value being between the minimum and maximum values. The result will default to a `bool` if no objects were specified through the `TrueObject` and/or `FalseObject` properties. If values are assigned to the `TrueObject` and `FalseObject` properties, the IsInRangeConverter returns the respective object assigned.
+
+> [!NOTE]
+> Note that the either **both** the `TrueObject` and `FalseObject` should have a value defined or **neither** should.
+
+The `ConvertBack` method is not supported.
+
+[!INCLUDE [common converter properties](../includes/communitytoolkit-converter.md)]
+
+## Syntax
+
+### XAML
+
+The `IsInRangeConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.IsInRangeConverterPage">
+        <ResourceDictionary>
+            <x:String x:Key="MinValue">Hello</x:String>
+            <x:String x:Key="MaxValue">World</x:String>
+            <Style x:Key="MyValueIsInRangeTrue" TargetType="Label">
+                <Setter Property="BackgroundColor" Value="Green" />
+            </Style>
+            <Style x:Key="MyValueIsInRangeFalse" TargetType="Label">
+                <Setter Property="BackgroundColor" Value="Red" />
+            </Style>
+        </ResourceDictionary>
+
+    <Label Style="{Binding MyValue, Converter={toolkit:IsInRangeConverter TrueObject={StaticResource MyValueIsInRangeTrue}, FalseObject={StaticResource MyValueIsInRangeFalse}, MinValue={StaticResource MinValue}, MaxValue={StaticResource MaxValue}}}" Text="The background of this label will be green if MyValue is between 'Hello' and 'World', otherwise red." />
+
+</ContentPage>
```

Comment:
Should this sample 'just' apply the `BackgroundColor` to the `Label` like the C# examples?

Reply:
Do you mean to set the `BackgroundColor` directly instead of using the style?
@GeorgeLeithead Please take a look at this comment and let's complete the docs :)

Reply:
Setting the property `BackgroundColor` (nor `Background`) doesn't work in XAML.
Using the Windows emulator, I tried to copy the `CompareConverter` page where the `BackgroundColor` is set with the converter.  The label background color does not change!
This also happens with  the `CompareConverter` examples, and MANY other examples listed in the documentation (when was the last time someone actually checked all the examples work?)

Reply:
Oh, looks like the `CompareConverter` has the same issue.
I guess it is probably the TrueObject and FalseObject don't get the "Green" or "Red" value as Color type?
We can then probably use the resources just like min and max values?
```
<ContentPage.Resources>
         . . .
        <Color x:Key="TrueColor">Green</Color>
        <Color x:Key="FalseColor">Red</Color>
    </ContentPage.Resources>
```
and use the values to the each objects?
```
<Label BackgroundColor="{Binding MyValue, Converter={toolkit:IsInRangeConverter TrueObject={StaticResource TrueColor}, FalseObject={StaticResource FalseColor}, MinValue={StaticResource MinValue}, MaxValue={StaticResource MaxValue}}}" Text="The background of this label will be green if MyValue is between 'Hello' and 'World', otherwise red." />
```

Reply:
It's NOT JUST `CompareConverter` that has the issue, I just highlighted this one, as I always use @bijington as inspiration for documentation, and this was his.
I'll have a look at using the `Color` type as soon as I can.

Reply:
All right, cool :) I will later also see if some sample code needs to be updated.

Reply:
Examples now use `Color` static resources and C# and XAML examples are aligned.

Reply:
Implemented to use `Color` static resource.

Reply:
Thanks for the update! :)

Reply:
@GeorgeLeithead great spot! I don't know how I messed that example up so much